src/map-marker/fixed-marker.js

```diff
@@ -125,10 +128,17 @@ const FixedMarker = ({
   }
 
   return (
-    <Root data-baseweb="fixed-map-marker" {...rootProps}>
+    <Root
+      data-baseweb="fixed-map-marker"
+      $isCentered={
+        size === PINHEAD_SIZES_SHAPES.xxSmallCircle ||
+        size === PINHEAD_SIZES_SHAPES.xxSmallSquare
+      }
+      {...rootProps}
+    >
       <FixedMarkerDragContainer
         $translateAmount={dragShadowMarginTop + dragShadowHeight}
-        $performTranslate={doesPinHeadTransformOnDrag && dragging}
+        $performTranslate={doesPinHeadTransformOnDrag && !dragging}
```

Comment:
The behavior of `performTranslate` was inverse which was confusing. I changed it to correspond to the name.